src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2861,11 +2861,11 @@ Right-click to remove the key combination, thereby deactivating the shortcut.</v
   </data>
   <data name="AlwaysOnTop.ModuleDescription" xml:space="preserve">
     <value>Always On Top is a quick and easy way to pin windows on top.</value>
-    <comment>"Always On Top" is the name of the utility</comment>
+    <comment>{Locked="Always On Top"}</comment>
   </data>
   <data name="AlwaysOnTop.ModuleTitle" xml:space="preserve">
     <value>Always On Top </value>
-    <comment>"Always On Top" is the name of the utility</comment>
+    <comment>{Locked="Always On Top"}</comment>
```

Comment:
if you are going to lock an entire resource (e.g. "don't change ANYTHING about this one") just use `{Locked}`